api/src/com/cloud/network/guru/NetworkGuruAdditionalFunctions.java

```diff
@@ -0,0 +1,12 @@
+package com.cloud.network.guru;
+
+import java.util.Map;
+
+public interface NetworkGuruAdditionalFunctions {
```

Comment:
Several problems with this interface. An interface called functions seems like it is meant o `patch up` another interface that is lacking functionality. Secondly, it looks very much NSX specific while the name and package seem quite generic.